core_backend/app/question_answer/config.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+# Functionality variables
+N_TOP_SIMILAR = os.environ.get("N_TOP_SIMILAR", "4")
```

Comment:
Took me a minute but I like your idea of moving config to within the submodule! Thanks for this.
This way we are less likely to have weird coupling if two submodules are using the same env var.